pdk/1.0/modules/ROOT/pages/policies-pdk-configure-network-proxy.adoc

```diff
@@ -0,0 +1,79 @@
+= Configuring a Network Proxy for PDK
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+If you use a network proxy to connect to the internet, it may be necessary to define the network proxy for the PDK tools.
+
+Besides Docker, all tools downloaded in the xref:policies-pdk-prerequisites.adoc[PDK prerequisites] accept the `HTTP_PROXY` and `HTTPS_PROXY` environment variables. 
+
+To configure your environment variable, refer to <<configure-a-network-proxy>>.
```

Comment:
Combine this lead-in with the next one. 
To configure your environment variable or use Docker with a network proxy...